app/utils.py

```diff
@@ -29,3 +30,54 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+def parse_to_db(db, filename):
+    city_default = ', philadelphia, pennsylvania, usa'
+    vehicle_id_index = 8
+    license_plate_index = 9
+    location_index = 4
+    date_index = 0
+    agency_index = 6
+    picture_index = 13
+    description_index = 11
+    with open(filename, 'rb') as file:
+        reader = csv.reader(file, delimiter=',')
+        columns = reader.next()
+        for row in reader:
+            address_text = row[location_index]
+            coordinates = geocoder.google(address_text + city_default).latlng
+            # Try arcgis geocode implementation if Google fails
+            if len(coordinates) == 0:
+                coordinates = geocoder.arcgis(address_text + city_default).latlng
+            loc = Location(
+                latitude=coordinates[0],
+                longitude=coordinates[1],
+                original_user_text=address_text)
+            db.session.add(loc)
+            date_format = "%m/%d/%Y %H:%M"
+            start_time = datetime.datetime.strptime(row[date_index], date_format)
+            end_time = datetime.datetime.strptime(row[date_index + 1], date_format)
+            # Assign correct agency id
+            agency_name = row[agency_index].rstrip().upper()
+            if agency_name == 'OTHER':
+                agency_name = row[agency_index + 1].rstrip().upper()
+            a = Agency.query.filter_by(name=agency_name).first()
+            if a is None:
+                a = Agency(name=agency_name)
+                a.is_public = False
+                a.is_official = False
+                db.session.add(a)
+                db.session.commit()
+            incident = IncidentReport(
+                vehicle_id=row[vehicle_id_index],
+                license_plate=row[license_plate_index],
```

Comment:
Are there any cells which are blank for `vehicle_id` or `license_plate`? If so, I would put `None` here instead of empty string to be consistent.